scaleway/resource_compute_instance_server.go

```diff
@@ -427,6 +429,7 @@ func resourceScalewayComputeInstanceServerUpdate(d *schema.ResourceData, m inter
 		if placementGroupID == "" {
 			updateRequest.ComputeCluster = &instance.NullableStringValue{Null: true}
 		} else {
+			forceReboot = true
```

Comment:
I think we need to do it just after the `if d.HasChange("placement_group_id")`.

Reply:
We could, but if we detach the instance from the placement group, we have no rules of placement so the current placement doesn't really matter anymore right? The reboot will be useless

Reply:
Ok